Modules/Filtering/Smoothing/include/itkDiscreteGaussianImageFilter.h

```diff
@@ -147,6 +152,10 @@ class ITK_TEMPLATE_EXPORT DiscreteGaussianImageFilter : public ImageToImageFilte
   itkSetMacro(RealBoundaryCondition, RealBoundaryConditionPointerType);
   itkGetConstMacro(RealBoundaryCondition, RealBoundaryConditionPointerType);
 
+  /** Get the variance, optionally adjusted for pixel spacing */
+  ArrayType
+  GetImageSpacingVarianceArray() const;
```

Comment:
Open to suggestions on a better name. 

This method returns the variance to use if image spacing is taken into consideration for the gaussian kernel. For each element,

` output[i] = variance[i] / ((image spacing [i]) ^2) `

Reply:
We may not want to expose this.

Reply:
As a `public` method at least.